nixos/doc/manual/release-notes/rl-1903.xml

```diff
@@ -408,6 +408,15 @@
        from nixpkgs due to the lack of maintainers.
      </para>
    </listitem>
+   <listitem>
+    <para>
+       The <option>powerManagement.cpuFreqGovernor</option> option has been
+       aliased to <option>powerManagement.cpufreq.governor</option>.  On laptops,
+       <option>powerManagement.cpuFreqGovernor</option> is sometimes set in
+       <literal>/etc/nixos/hardware.nix</literal>, so you can rename it to the new
```

Comment:
s/hardware.nix/hardware-configuration.nix

Reply:
Sorry, I thought I had fixed that in a previous rebase.

Fixed in commit b0f10d2d531.